bundles/org.openhab.binding.telegram/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -55,6 +56,11 @@
 				<default>SOCKS5</default>
 				<description>Enter your proxy type. Default: SOCKS5</description>
 			</parameter>
+			<parameter name="longPollingTime" type="integer" required="true" min="0" max="50" unit="s">
```

Comment:
Why is the parameter set to "required"? I don't think this is necessary (since we also have a default value) and I think we would have issues with backward compatibility (an update of the Telegram component would break an existing configuration).

Reply:
Good idea. I will fix it.